<?xml version="1.0" encoding="UTF-8"?>
<ui version="6.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GESTION DES NOTES</string>
  </property>
  <property name="styleSheet">
              <string>
                  QWidget {
                      background-color: #5784BA;
                      color: #212E53;
                  }
                  QLineEdit {
                      background-color: #BED3C3;
                      color: #212E53;
                      border: 1px solid #5784BA;
                  }
                  QPushButton {
                      background-color: #5784BA;
                      border: 1px solid #212E53;
                      color: #212E53;
                      border-radius: 3px;
                      padding: 5px;
                  }
                  QPushButton:hover {
                      background-color: #BED3C3;
                  }
                  QPushButton:pressed {
                      background-color: #212E53;
                  }
              </string>
          </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>301</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Connexion</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="Username">
        <property name="text">
         <string>Nom d'utilisateur</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Username"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="Password">
        <property name="text">
         <string>Mot de passe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Password">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Se connecter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_Login">
      <property name="text">
       <string>Statut : </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>146</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>  Statut ++ </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Picture2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string> TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
